admin_guide/out_of_resource_handling.adoc

```diff
@@ -12,39 +12,161 @@ toc::[]
+
+By using well-configured eviction policies, a node can proactively monitor for and prevent
+against total starvation of a compute resource. 
+
+For more information on eviction trigger signals, see xref:out-of-resource-eviction-signals[Understanding Eviction Signals] below.
 
-Using eviction policies, a node can proactively monitor for and prevent
-against total starvation of a compute resource.
+For more information on eviction thresholds, see xref:out-of-resource-eviction-thresholds[Understanding Eviction Thresholds] below.
 
-In cases where a node is running low on available resources, it can proactively
-fail one or more pods in order to reclaim the starved resource using an eviction
-policy. When the node fails a pod, it terminates all containers in the pod, and
+[NOTE]
+====
+When the node fails a pod, it terminates all containers in the pod, and
 the `PodPhase` is transitioned to *Failed*.
+====
+
+[[out-of-resource-create-config]]
+=== Using the Node Configuration to Create a Policy 
+
+To configure an eviction policy using the node configuration file:
+
+. Edit the node configuration file on a node.
+
+. Add or edit the eviction thresholds under the `eviction-hard` or `eviction-soft` paramters:
++
+For example:
++
+.Sample Node Configuration file for a hard eviction 
+====
+----
+kubeletArguments:
+  eviction-hard: <1>
+  - memory.available<500Mi<2>
+  - nodefs.available<500Mi
+  - nodefs.inodesFree<100Mi
+  - imagefs.available<100Mi
+  - imagefs.inodesFree<100Mi
+----
+
+<1> The type of eviction: Use this parameter for a xref:out-of-resource-hard-eviction-thresholds[hard eviction].
+<2> An eviction threshold based on a specific eviction trigger signal.
+====
++
+.Sample Node Configuration file for a soft eviction
+====
+----
+kubeletArguments:
+  eviction-soft: <1>
+  - memory.available<500Mi <2>
+  - nodefs.available<500Mi
+  - nodefs.inodesFree<100Mi
+  - imagefs.available<100Mi
+  - imagefs.inodesFree<100Mi
+  eviction-soft-grace-period:<3>
+  - memory.available=1m30s
+  - nodefs.available=1m30s
+  - nodefs.inodesFree=1m30s
+  - imagefs.available=1m30s
+  - imagefs.inodesFree=1m30s
+----
+
+<1> The type of eviction: Use this parameter for a xref:out-of-resource-hard-eviction-thresholds[soft eviction].
+<2> An eviction threshold based on a specific eviction trigger signal.
+<3> The grace period for the soft eviction. Leave the default values for best performance.
+====
+
+. Restart the {product-title} services for the changes to take effect:
++
+ifdef::openshift-enterprise[]
+----
+# systemctl restart atomic-openshift-node
+----
+endif::[]
+ifdef::openshift-origin[]
+----
+# systemctl restart origin-node
+----
+endif::[]
 
-Platform administrators can configure eviction settings within the
-*_node-config.yaml_* file.
+//
+[[out-of-resource-create-cli]]
+=== Using the Node CLI to Create a Policy
```

Comment:
is the comment hiding this?  we don't have a node cli so this section doesnt make much sense in openshift context.